pymlx/requirements.txt

```diff
@@ -4,3 +4,5 @@ pandas>=0.15.0
 scikit_learn>=0.15.0
 matplotlib>=1.4.3
 dill>=0.2.2
+lightgbm
```

Comment:
Let's remove specific ML toolkits in the requirements. If users want to use `lightgbm`, he/she needs to install it manually.